api/src/main/resources/messages.properties

```diff
@@ -31,6 +31,7 @@ coreapps.age.days={0} Day(s)
 coreapps.patientDashBoard.editPatientIdentifier.duplicateMessage=is already in use
 coreapps.patientDashBoard.editPatientIdentifier.invalidMessage=did not pass validation
 coreapps.patientDashBoard.editPatientIdentifier.invalidFormatMessage=must be in the format
+coreapps.patientdashboard.activeDrugOrders=Dispensed Medication
```

Comment:
Not sure if this should be plural.

Reply:
@mks-d, are you referring to something like `Dispensed Medications`?

Reply:
Yes but no ;-) This word stays singular in English. We keep it as it is.